### ISSUE TITLE
Levelup monotone display

### ISSUE THREAD

**Issue by theriex** [theriex/rh#14]
If all points in a timeline have the same question type and don't differentiate based on group or tag values, then the levelup display doesn't show anything interesting for a background.  It's basically charting one value at 100% which looks like nothing.  It's correct in showing it is a homogenous timeline, but it's visually bad.

A second issue is that levelup.makeStackData does not plot every point.  Instead, it groups points into ncxt size chunks to create a display that is not crazy jagged and unresolved across clusters of points (like if there are many more points in recent time).  Because of this grouping, adding more fields to the mix such as text word count, number of references, pic included, time spent on the point etc might help.  It won't necessarily alleviate the issue, but probably worth trying to see if it's enough.

Meanwhile try and use some different question types, keywords and tags.

**Comment by theriex** [comment]
Grouping points for the stacked area chart makes sense.  Want 12-24 total points, ok to put any leftover points ungrouped to the right.  Want some vaguely meaningful fields so it shows some vaguely interesting information: time/pt, numrefs, generation, groups, regions, categories, tags

**Comment by theriex** [comment]
It's a good idea to vary the question types to make the interactive traversal more interesting, and I've added "date required" and "lesser known" counts to the general timeline informational line when editing (along with the total number of points and pics).  More question variety will help create an interesting stacked area background on levelup, but it's not enough by itself.

It is also possible to have timelines where the groups, regions, categories and tags are the same for all points.  When it varies, high difference values feed into the levelup stacked area background for interest, but this is also not enough by itself.

Scalar values of point text length, how many references, and interaction time are always available and have enough data to create a reasonable background display.  Added those to the levelup processing which provides a lot more to work with.  The resulting backgrounds are more interesting, and should work across a wide variety of timelines.
